.circleci/config.yml

```diff
@@ -221,6 +221,9 @@ commands:
          name: Install postgresql client
          command: |
            sudo rm -rf /var/lib/apt/lists/*
+           # heroku.com is not updating their keys
```

Comment:
Any idea why this is not needed in dockstore/dockstore or is this about to fail there too?
`cimg/openjdk` is slightly older but the machine image appears to be the same

Reply:
I do not know why this is not needed in dockstore/dockstore